packages/react-native-gesture-handler/src/components/GestureComponents.web.tsx

```diff
@@ -31,7 +31,7 @@ export const DrawerLayoutAndroid = () => {
 export const RefreshControl = createNativeWrapper(View);
 
 export const FlatList = React.forwardRef(
-  <ItemT extends any>(props: FlatListProps<ItemT>, ref: any) => (
+  <ItemT,>(props: FlatListProps<ItemT>, ref: any) => (
```

Comment:
Why is a comma there?

Reply:
Because if you don't have comma, it will be treated as `tag` instead of type.